server/src/main/java/au/org/aodn/ogcapi/server/common/RestApi.java

```diff
@@ -55,6 +57,9 @@ public class RestApi implements ApiApi, DefaultApi, ConformanceApi {
     @Autowired
     protected StacToCollections stacToCollection;
 
+    @Autowired
```

Comment:
I think this will be better to isolate it to another class said CommonRestExtApi etc so that changes in the OGC api will not messy up with our extension.